### ISSUE TITLE
ImportError: cannot import name 'plot' from 'continuum.viz'

### ISSUE THREAD

**Issue by lebrice** [Continvvm/continuum#152]
```python3
from continuum.task_set import TaskSet
```
```console
continuum_bug.py:1: in <module>
    from continuum.task_set import TaskSet
../../miniconda3/envs/sequoia/lib/python3.8/site-packages/continuum/task_set.py:8: in <module>
    from continuum.viz import plot
E   ImportError: cannot import name 'plot' from 'continuum.viz' (/home/fabrice/miniconda3/envs/sequoia/lib/python3.8/site-packages/continuum/viz.py)
```
Happens with continuum version: `1.0.19` as well as `1.0.25`

Is this `task_set.py` the same as `continuum.tasks.task_set.py` ? If so, maybe removing it would be good?

**Comment by lebrice** [comment]
Had a little chat with @TLESORT , I think I know what the issue is:
It seems to me like this is due to the old file still being present in your `build` folder when you're manually creating a PyPI release!
Simplest solution would be to clear the folder, but the best solution would be to switch to using Travis to automate deploying to PyPI!

Let me know if you need any help setting this up, I've already done this for another library of mine.

**Comment by arthurdouillard** [comment]
I've fixed it in 1.026. I forgot to clear the build/ folder in my makefile. Thanks @lebrice!

About automatic deploying, I'll look into it. I'm reading the page about encrypting my api token. If I have questions, I'll reach out to you on slack :)
